### PR TITLE
Update step/stage terminology, refactor v1 Scanner Reference topics STO-7458

### PR DESCRIPTION
**Pull request by douglas-j-bothwell**
### Changes in this PR
- Update all **Security Tests stage** references to **Security stage**
- Update all **Security step** references to 
  - **Custom Scan step** for scanners without step palettes
  - **Bandit step**, **Semgrep step**, etc. for scanners with step palettes
  - **scan step** or **Security Test step** to refer generically to any STO scan step
- Refactor v1 scanner references (ie integrations based on Custom Scan step)
   - Streamline/simplify Important Notes
   - Better support for copy-pasting YAML step definitions and strings such as ingestionOnly
   - Add button links at the top of each reference so you can immediately see the target types and ingestion modes supported for each scanner

#### [Custom Scan step reference](https://664a55b389a77bcac8268550--harness-developer.netlify.app/docs/security-testing-orchestration/sto-techref-category/custom-scan-reference) 

This topic includes all the snippets used in the other Custom Scan steps. 

List of all non-step-palette scanner integrations:
- [_scanners-that-use-custom-scan-step.md](https://github.com/harness/developer-hub/pull/6410/files#diff-4427a19c35260095612b6ebf8bfcdabadb38d71762ab1330cd65db6e848e9074)

Workflow snippets:
- [_workflow-ingest-only.md](https://github.com/harness/developer-hub/pull/6410/files#diff-f92163f61a0d58bd2788edfa5b97128aa949b971474fc2f859bb50ad07e4b848) For scanners that only support ingestion mode
- [_workflow-repo.md](https://github.com/harness/developer-hub/pull/6410/files#diff-a3a517acf3d3e4c112d6ad1decf6a74ef616b59f978c2a4a42a90fcf0670ce3f) For orchestration/extraction scans of code repos. Includes a step to configure the codebase.
- [_workflow.md](https://github.com/harness/developer-hub/pull/6410/files#diff-321ccaef2ee3b868dde1cfe635a7e4b003cf5950c7c10cabe23ba23b1239b3b5) A generic snippet for all other workflows:

Settings snippets:
   - [_config.md](https://github.com/harness/developer-hub/pull/6410/files#diff-fcdb0f9f562b5244e114c9cdde49482c6cd72f0ea6e4c09b4a34d8a09be30600)
   - [_container.md](https://github.com/harness/developer-hub/pull/6410/files#diff-d645131ad7153c60f6979d52481c8b8da1f3fb0112f6d56535fb2bcae5439972)
   - [_dast.md](https://github.com/harness/developer-hub/pull/6410/files#diff-5c81d26c2486bcaa948f4f97274a7cbc36ee1cdd1f702dea09816ef188584a17)
   - [_fail-on-severity.md](https://github.com/harness/developer-hub/pull/6410/files#diff-e06317473745e42e44b7d1d3f05cad34366dc776a7a42214dd415a92760dc191)
   - [_ingestion-file.md](https://github.com/harness/developer-hub/pull/6410/files#diff-4f326a09d31540f676564b7cdf41aa0de2b7367311c17f043793350bc5c924b1)
   - [_repo.md](https://github.com/harness/developer-hub/pull/6410/files#diff-04ea1ef3a04ccc0a818eac3b3b04190cec180c18552b8e9e37ea222f83e9836a)
   - [_scanners-that-use-custom-scan-step.md](https://github.com/harness/developer-hub/pull/6410/files#diff-4427a19c35260095612b6ebf8bfcdabadb38d71762ab1330cd65db6e848e9074)
   - [_target-variant.md](https://github.com/harness/developer-hub/pull/6410/files#diff-6ca032761f80c40627afbd16d414dff515eeb151a32a9ed033fe7762793efdf6)

#### Custom Scan reference topics organized by workflow type

- [_workflow-ingest-only.md](https://github.com/harness/developer-hub/pull/6410/files#diff-f92163f61a0d58bd2788edfa5b97128aa949b971474fc2f859bb50ad07e4b848) For scanners that only support ingestion mode:
   - [Fortify Static Code Analyzer scanner reference](https://664a55b389a77bcac8268550--harness-developer.netlify.app/docs/security-testing-orchestration/sto-techref-category/fortify-scanner-reference)
   - [HCL AppScan scanner reference](https://664a55b389a77bcac8268550--harness-developer.netlify.app/docs/security-testing-orchestration/sto-techref-category/hql-appscan-scanner-reference)
   - [Qualys Web Application Scanning (WAS) scanner reference](https://664a55b389a77bcac8268550--harness-developer.netlify.app/docs/security-testing-orchestration/sto-techref-category/qualys-web-app-scanner-reference)
   - [Reapsaw scanner reference scanner reference](https://664a55b389a77bcac8268550--harness-developer.netlify.app/docs/security-testing-orchestration/sto-techref-category/reapsaw-scanner-reference)

- [_workflow-repo.md](https://github.com/harness/developer-hub/pull/6410/files#diff-a3a517acf3d3e4c112d6ad1decf6a74ef616b59f978c2a4a42a90fcf0670ce3f) For orchestration/extraction scans of code repos. Includes a step to configure the codebase.
   - [Data Theorem scanner reference](https://664a55b389a77bcac8268550--harness-developer.netlify.app/docs/security-testing-orchestration/sto-techref-category/data-theorem-scanner-reference)
   - [Fortify on Demand scanner reference](https://664a55b389a77bcac8268550--harness-developer.netlify.app/docs/security-testing-orchestration/sto-techref-category/fortify-on-demand-scanner-reference)
   - [Nexus scanner reference](https://664a55b389a77bcac8268550--harness-developer.netlify.app/docs/security-testing-orchestration/sto-techref-category/nexus-scanner-reference)
   - [Qwiet AI (formerly ShiftLeft) scanner reference](https://664a55b389a77bcac8268550--harness-developer.netlify.app/docs/security-testing-orchestration/sto-techref-category/qwiet-scanner-reference)
   - [Veracode scanner reference](https://664a55b389a77bcac8268550--harness-developer.netlify.app/docs/security-testing-orchestration/sto-techref-category/veracode-scanner-reference)

- [_workflow.md](https://github.com/harness/developer-hub/pull/6410/files#diff-321ccaef2ee3b868dde1cfe635a7e4b003cf5950c7c10cabe23ba23b1239b3b5) A generic snippet for all other workflows:
   - [Clair scanner reference](https://664a55b389a77bcac8268550--harness-developer.netlify.app/docs/security-testing-orchestration/sto-techref-category/clair-scanner-reference)
   - [Docker Content Trust (DCT) scanner reference](https://664a55b389a77bcac8268550--harness-developer.netlify.app/docs/security-testing-orchestration/sto-techref-category/docker-content-trust-dct-scanner-reference)
   - [Metasploit Framework scanner reference](https://664a55b389a77bcac8268550--harness-developer.netlify.app/docs/security-testing-orchestration/sto-techref-category/metasploit-scanner-reference)
   - [Nessus scanner reference](https://664a55b389a77bcac8268550--harness-developer.netlify.app/docs/security-testing-orchestration/sto-techref-category/nessus-scanner-reference)
   - [OpenVAS scanner reference](https://664a55b389a77bcac8268550--harness-developer.netlify.app/docs/security-testing-orchestration/sto-techref-category/openvas-scanner-reference)
   - [ScoutSuite scanner reference](https://664a55b389a77bcac8268550--harness-developer.netlify.app/docs/security-testing-orchestration/sto-techref-category/scoutsuite-scanner-reference)
   - [Tenable scanner reference](https://664a55b389a77bcac8268550--harness-developer.netlify.app/docs/security-testing-orchestration/sto-techref-category/tenable-scanner-reference)

### Scanner and Product Access settings

These links point to the **Scanner settings** and **Product access** sections that are "hard-coded" i.e not imported snippets. Each updated topic includes a (_legacy_) link that points to the earliest committed version of the topic (i.e., when the content first got ported over to the HDH). These legacy links make it easier to cross-check the current content against the legacy content.  

1. [Clair scanner reference](https://664a55b389a77bcac8268550--harness-developer.netlify.app/docs/security-testing-orchestration/sto-techref-category/clair-scanner-reference)      {[_HDH_](https://developer.harness.io/docs/security-testing-orchestration/sto-techref-category/clair-scanner-reference#clair-scan-settings)
   a. [Scanner settings](https://664a55b389a77bcac8268550--harness-developer.netlify.app/docs/security-testing-orchestration/sto-techref-category/clair-scanner-reference/#scanner-settings)
2.  [Data Theorem](https://664a55b389a77bcac8268550--harness-developer.netlify.app/docs/security-testing-orchestration/sto-techref-category/data-theorem-scanner-reference)    ([_HDH_](https://developer.harness.io/docs/security-testing-orchestration/sto-techref-category/data-theorem-scanner-reference#data-theorem-scan-settings))
   a. [Scanner settings](https://664a55b389a77bcac8268550--harness-developer.netlify.app/docs/security-testing-orchestration/sto-techref-category/data-theorem-scanner-reference/#scanner-settings)
   b. [Product access](https://664a55b389a77bcac8268550--harness-developer.netlify.app/docs/security-testing-orchestration/sto-techref-category/data-theorem-scanner-reference/#product-access)
2.  [Docker Content Trust](https://664a55b389a77bcac8268550--harness-developer.netlify.app/docs/security-testing-orchestration/sto-techref-category/docker-content-trust-dct-scanner-reference)    ([_HDH_](https://developer.harness.io/docs/security-testing-orchestration/sto-techref-category/docker-content-trust-dct-scanner-reference#docker-content-trust-dct-scan-settings))
   a. [Scanner settings](https://664a55b389a77bcac8268550--harness-developer.netlify.app/docs/security-testing-orchestration/sto-techref-category/docker-content-trust-dct-scanner-reference/#scanner-settings)
2.  [Fortify Static Code Analyzer scanner reference](https://664a55b389a77bcac8268550--harness-developer.netlify.app/docs/security-testing-orchestration/sto-techref-category/fortify-scanner-reference)     ([_HDH_](https://developer.harness.io/docs/security-testing-orchestration/sto-techref-category/fortify-scanner-reference#fortify-scan-settings))
   a. [Scanner settings](https://664a55b389a77bcac8268550--harness-developer.netlify.app/docs/security-testing-orchestration/sto-techref-category/fortify-scanner-reference#scanner-settings)
2.  [Fortify on Demand](https://664a55b389a77bcac8268550--harness-developer.netlify.app/docs/security-testing-orchestration/sto-techref-category/fortify-on-demand-scanner-reference)     ([_HDH_](https://developer.harness.io/docs/security-testing-orchestration/sto-techref-category/fortify-on-demand-scanner-reference#fortify-on-demand-scan-settings))
   a. [Scanner settings](https://664a55b389a77bcac8268550--harness-developer.netlify.app/docs/security-testing-orchestration/sto-techref-category/fortify-on-demand-scanner-reference/#scanner-settings)
   b. [Product access](https://664a55b389a77bcac8268550--harness-developer.netlify.app/docs/security-testing-orchestration/sto-techref-category/fortify-on-demand-scanner-reference/#product-access)
2.  [HCL AppScan](https://664a55b389a77bcac8268550--harness-developer.netlify.app/docs/security-testing-orchestration/sto-techref-category/hql-appscan-scanner-reference)
   a.  [Scanner settings](https://664a55b389a77bcac8268550--harness-developer.netlify.app/docs/security-testing-orchestration/sto-techref-category/hql-appscan-scanner-reference/#scanner-settings)
2.  [Metasploit](https://664a55b389a77bcac8268550--harness-developer.netlify.app/docs/security-testing-orchestration/sto-techref-category/metasploit-scanner-reference)    ([_HDH_](https://developer.harness.io/docs/security-testing-orchestration/sto-techref-category/metasploit-scanner-reference#metasploit-scan-settings))
   a. [Scanner settings](https://664a55b389a77bcac8268550--harness-developer.netlify.app/docs/security-testing-orchestration/sto-techref-category/metasploit-scanner-reference/#scanner-settings)
2.  [Nessus](https://664a55b389a77bcac8268550--harness-developer.netlify.app/docs/security-testing-orchestration/sto-techref-category/nessus-scanner-reference)     ([_HDH_](https://developer.harness.io/docs/security-testing-orchestration/sto-techref-category/nessus-scanner-reference#nessus-scan-settings))
   a. [Scanner settings](https://664a55b389a77bcac8268550--harness-developer.netlify.app/docs/security-testing-orchestration/sto-techref-category/nessus-scanner-reference/#scanner-settings)
   b. [Product access](https://664a55b389a77bcac8268550--harness-developer.netlify.app/docs/security-testing-orchestration/sto-techref-category/nessus-scanner-reference/#product-access)
2.  [Nexus](https://664a55b389a77bcac8268550--harness-developer.netlify.app/docs/security-testing-orchestration/sto-techref-category/nexus-scanner-reference)    ([_HDH_](https://developer.harness.io/docs/security-testing-orchestration/sto-techref-category/nexus-scanner-reference#nexus-scan-settings))
   a. [Scanner settings](https://664a55b389a77bcac8268550--harness-developer.netlify.app/docs/security-testing-orchestration/sto-techref-category/nexus-scanner-reference/#scanner-settings)
   b. [Product access](https://664a55b389a77bcac8268550--harness-developer.netlify.app/docs/security-testing-orchestration/sto-techref-category/nexus-scanner-reference/#product-access)
2.  [OpenVAS](https://664a55b389a77bcac8268550--harness-developer.netlify.app/docs/security-testing-orchestration/sto-techref-category/openvas-scanner-reference)    ([_HDH_](https://developer.harness.io/docs/security-testing-orchestration/sto-techref-category/openvas-scanner-reference#openvas-scan-settings))
   a. [Scanner settings](https://664a55b389a77bcac8268550--harness-developer.netlify.app/docs/security-testing-orchestration/sto-techref-category/openvas-scanner-reference/#scanner-settings)
   b. [Product access](https://664a55b389a77bcac8268550--harness-developer.netlify.app/docs/security-testing-orchestration/sto-techref-category/openvas-scanner-reference/#product-access)
2.  [Qualys Web Application Scanning](https://664a55b389a77bcac8268550--harness-developer.netlify.app/docs/security-testing-orchestration/sto-techref-category/qualys-web-app-scanner-reference)     ([_HDH_](https://developer.harness.io/docs/security-testing-orchestration/sto-techref-category/qualys-web-app-scanner-reference#qualys-scan-settings))
   a. [Scanner settings](https://664a55b389a77bcac8268550--harness-developer.netlify.app/docs/security-testing-orchestration/sto-techref-category/qualys-web-app-scanner-reference/#scanner-settings)
2.  [Qwiet AI (formerly ShiftLeft)](https://664a55b389a77bcac8268550--harness-developer.netlify.app/docs/security-testing-orchestration/sto-techref-category/qwiet-scanner-reference)     ([_HDH_](https://developer.harness.io/docs/security-testing-orchestration/sto-techref-category/qwiet-scanner-reference#qwiet-ai-scan-settings))
   a. [Scanner settings](https://664a55b389a77bcac8268550--harness-developer.netlify.app/docs/security-testing-orchestration/sto-techref-category/qwiet-scanner-reference/#scanner-settings)
   b. [Product access](https://664a55b389a77bcac8268550--harness-developer.netlify.app/docs/security-testing-orchestration/sto-techref-category/qwiet-scanner-reference/#product-access)
2.  [Reapsaw](https://664a55b389a77bcac8268550--harness-developer.netlify.app/docs/security-testing-orchestration/sto-techref-category/reapsaw-scanner-reference)    ([_HDH_](https://developer.harness.io/docs/security-testing-orchestration/sto-techref-category/reapsaw-scanner-reference#reapsaw-scan-settings))
   a. [Scanner settings](https://664a55b389a77bcac8268550--harness-developer.netlify.app/docs/security-testing-orchestration/sto-techref-category/reapsaw-scanner-reference/#scanner-settings)
2.  [ScoutSuite](https://664a55b389a77bcac8268550--harness-developer.netlify.app/docs/security-testing-orchestration/sto-techref-category/scoutsuite-scanner-reference)    ([_HDH_](https://developer.harness.io/docs/security-testing-orchestration/sto-techref-category/scoutsuite-scanner-reference#scoutsuite-scan-settings))
   a. [Scanner settings](https://664a55b389a77bcac8268550--harness-developer.netlify.app/docs/security-testing-orchestration/sto-techref-category/scoutsuite-scanner-reference/#scanner-settings)
2.  [Tenable](https://664a55b389a77bcac8268550--harness-developer.netlify.app/docs/security-testing-orchestration/sto-techref-category/tenable-scanner-reference)     ([_HDH_](https://developer.harness.io/docs/security-testing-orchestration/sto-techref-category/tenable-scanner-reference#tenable-settings))
   a. [Scanner settings](https://664a55b389a77bcac8268550--harness-developer.netlify.app/docs/security-testing-orchestration/sto-techref-category/tenable-scanner-reference/#scanner-settings)
   b. [Product access](https://664a55b389a77bcac8268550--harness-developer.netlify.app/docs/security-testing-orchestration/sto-techref-category/tenable-scanner-reference/#product-access)
2.  [Veracode](https://664a55b389a77bcac8268550--harness-developer.netlify.app/docs/security-testing-orchestration/sto-techref-category/veracode-scanner-reference)     ([_HDH_](https://developer.harness.io/docs/security-testing-orchestration/sto-techref-category/veracode-scanner-reference#required-settings-for-veracode-scans-in-sto))
   a. [Scanner settings](https://664a55b389a77bcac8268550--harness-developer.netlify.app/docs/security-testing-orchestration/sto-techref-category/veracode-scanner-reference/#scanner-settings)
2.  [JFrog Xray](https://664a55b389a77bcac8268550--harness-developer.netlify.app/docs/security-testing-orchestration/sto-techref-category/xray-scanner-reference)     ([_HDH_](https://developer.harness.io/docs/security-testing-orchestration/sto-techref-category/xray-scanner-reference#jfrog-xray-settings))
   a. [Scanner settings](https://664a55b389a77bcac8268550--harness-developer.netlify.app/docs/security-testing-orchestration/sto-techref-category/xray-scanner-reference#scanner-settings)

### Prepub checks

PRs must meet these requirements to be merged:

- [x] Successful preview build.
- [x] SME review.
- [x] Editorial review.
- [x] PM review (went over with PC in our weekly 1:1)
- [x] No merge conflicts.
- [ ] Release notes/new features docs: Feature/version released to at least one prod environment.

### CODE REVIEW

File: docs/security-testing-orchestration/sto-techref-category/aquasec-scanner-reference.md [hunk 1]
```diff
@@ -15,31 +21,25 @@ The Aqua Security step can also ingest assurance policy violations. These violat
 
 ## Important notes for running Aqua Security scans in STO
 
+- You need to add a [Docker-in-Docker background step](/docs/security-testing-orchestration/sto-techref-category/security-step-settings-reference#docker-in-docker-requirements-for-sto) to run orchestration scans on Kubernetes or Docker build infrastructures.
```
Comment (SmritiSatya) [inline docs/security-testing-orchestration/sto-techref-category/aquasec-scanner-reference.md:24]: You need to:
- Add a [Docker-in-Docker background step](/docs/security-testing-orchestration/sto-techref-category/security-step-settings-reference#docker-in-docker-requirements-for-sto) to run orchestration scans on Kubernetes or Docker build infrastructures.
- Run the scan step with root access if either of the following apply:
- Run a [Docker-in-Docker background service](/docs/security-testing-orchestration/sto-techref-category/security-step-settings-reference/#docker-in-docker-requirements-for-sto).
- Add trusted certificates to your scan images at runtime.

File: docs/security-testing-orchestration/sto-techref-category/aws-ecr-scanner-reference.md [hunk 2]
```diff
@@ -5,24 +5,26 @@ sidebar_label: AWS ECR scanner reference
...
+<DocsTag  text="Artifact scanners" backgroundColor= "#cbe2f9" textColor="#0b5cad" link="/docs/security-testing-orchestration/sto-techref-category/security-step-settings-reference#artifact-scanners"  />
+<DocsTag  text="Extraction" link="/docs/security-testing-orchestration/use-sto/orchestrate-and-ingest/sto-workflows-overview/#extraction-scans-in-sto" />
+<br/>
+<br/>
 
-## Important notes for running AWS ECR scans in STO
+You can scan your container images and extract scan results from [Amazon Elastic Container Registry (ECR)](https://docs.aws.amazon.com/AmazonECR/latest/userguide/what-is-ecr.html). 
 
-### Docker-in-Docker requirements
+## Important notes for running AWS ECR scans in STO
```
Comment (SmritiSatya) [inline docs/security-testing-orchestration/sto-techref-category/aws-ecr-scanner-reference.md:16]: Global comment
You need to:
- ...

File: docs/security-testing-orchestration/sto-techref-category/clair-scanner-reference.md [hunk 3]
```diff
@@ -5,40 +5,121 @@ sidebar_label: Clair scanner reference
...
+- [Scan type](#scan-type)
+- [Policy type](#policy-type)
+- [Product config name](#product-config-name)
 
 
-### For more information
+#### Product name
 
-import StoMoreInfo from '/docs/security-testing-orchestration/sto-techref-category/shared/_more-information.md';
+The scanner name. Required for all Custom Scan steps.
```
Comment (SmritiSatya) [inline docs/security-testing-orchestration/sto-techref-category/clair-scanner-reference.md:35]: Global
Scanner name (required for all Custom Scan steps).
OR
Scanner name that is required for all Custom Scan steps.

File: docs/security-testing-orchestration/sto-techref-category/nexus-scanner-reference.md [hunk 4]
```diff
@@ -59,18 +126,128 @@ import StoLegacyTargetAndVariant  from './shared/legacy/_sto-ref-legacy-target-a
...
+##### Key
+
+```
+product_access_id
+```
+
+##### Value
+
+The Harness secret for the  password used to log in to the NexusIQ UI.
+
```
Comment (SmritiSatya) [inline docs/security-testing-orchestration/sto-techref-category/nexus-scanner-reference.md:163]: The Harness secret for the password used to log .. 
(There was an extra space between 'the' and 'password')

File: docs/security-testing-orchestration/sto-techref-category/shared/custom-scan/_repo.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+repository_project
+```
+###### Value
+
+The name of the repo to scan. To specify the repo URL, edit  the [Codebase Config object](/docs/continuous-integration/use-ci/codebase-configuration/create-and-configure-a-codebase) in the Harness pipeline.  
+
+In most cases, this should match the repo name used in your Git provider.
+
+#### Repository Branch
+
```
Comment (SmritiSatya) [inline docs/security-testing-orchestration/sto-techref-category/shared/custom-scan/_repo.md:26]: Repository branch

File: docs/security-testing-orchestration/sto-techref-category/shared/custom-scan/_repo.md [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+<!-- TOC start (generated with https://github.com/derlin/bitdowntoc) -->
+
+- [Repository name](#repository-name)
+- [Repository branch](#variant)
+
+<!-- TOC end -->
+
+#### Repository Project
+
```
Comment (SmritiSatya) [inline docs/security-testing-orchestration/sto-techref-category/shared/custom-scan/_repo.md:13]: Repository project

File: docs/security-testing-orchestration/sto-techref-category/shared/custom-scan/_target-variant.md [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+```yaml
+target_name
+```
+
+###### Value
+A user-defined label for the code repository, container, application, or configuration to scan. Specify a unique, descriptive name. This makes it much easier to navigate your scan results in the STO UI.
+
+
+#### Target Variant
+
```
Comment (SmritiSatya) [inline docs/security-testing-orchestration/sto-techref-category/shared/custom-scan/_target-variant.md:23]: Target variant

**Review by SmritiSatya — APPROVED** [review]

**Review by SmritiSatya — DISMISSED** [review]

File: docs/security-testing-orchestration/sto-techref-category/custom-scan-reference.md [hunk 8]
```diff
@@ -0,0 +1,250 @@
...
+
+<CustomScanTargetVariant />
+
+### Code repositories 
+
+import CustomScanRepo from './shared/custom-scan/_repo.md';
+
+<CustomScanRepo />
+
+### Container images
```
Comment (aklaft) [inline docs/security-testing-orchestration/sto-techref-category/custom-scan-reference.md:201]: are these artifact scans now?

**Review by aklaft — COMMENTED** [review]
left a couple comments. I think I'll have to reapprove anyway, but if you look at these I think it's good.

File: docs/security-testing-orchestration/sto-techref-category/custom-scan-reference.md [hunk 9]
```diff
@@ -0,0 +1,250 @@
...
+This workflow applies to scanner integrations that support Ingestion mode only.
+
+import CustomScanWorkflowRepo from './shared/custom-scan/_workflow-repo.md';
+
+<CustomScanWorkflowRepo />
+
+</details>
+
+<details>
+<summary>All other workflows</summary>
```
[the remark this answers is not in the document]
Comment (douglas-j-bothwell) [reply docs/security-testing-orchestration/sto-techref-category/custom-scan-reference.md]: @aklaft I refactored these workflow descriptions into an ingestion workflow and an orchestration/extraction workflow: [Workflow descriptions](https://664a3ca589a77bb6052686d3--harness-developer.netlify.app/docs/security-testing-orchestration/sto-techref-category/custom-scan-reference#workflow-descriptions)

File: docs/security-testing-orchestration/sto-techref-category/custom-scan-reference.md [hunk 10]
```diff
@@ -0,0 +1,250 @@
...
+
+### Code repositories 
+
+import CustomScanRepo from './shared/custom-scan/_repo.md';
+
+<CustomScanRepo />
+
+### Container images 
+
+import CustomScanContainer from './shared/custom-scan/_container.md';
```
[the remark this answers is not in the document]
Comment (douglas-j-bothwell) [reply docs/security-testing-orchestration/sto-techref-category/custom-scan-reference.md:188]: @aklaft  Updated these descriptions with info from the tooltips [Artifacts](https://664a3ca589a77bb6052686d3--harness-developer.netlify.app/docs/security-testing-orchestration/sto-techref-category/custom-scan-reference#artifacts)

File: docs/security-testing-orchestration/sto-techref-category/custom-scan-reference.md [hunk 11]
```diff
@@ -0,0 +1,250 @@
...
+
+
+## Additional Configuration
+
+In the **Additional Configuration** settings, you can use the following options:
+
+* [Privileged](/docs/continuous-integration/use-ci/manage-dependencies/background-step-settings#privileged)
+* [Image Pull Policy](/docs/continuous-integration/use-ci/manage-dependencies/background-step-settings#image-pull-policy)
+* [Run as User](/docs/continuous-integration/use-ci/manage-dependencies/background-step-settings#run-as-user)
+* [Set Container Resources](/docs/continuous-integration/use-ci/manage-dependencies/background-step-settings#set-container-resources)
```
[the remark this answers is not in the document]
Comment (douglas-j-bothwell) [reply docs/security-testing-orchestration/sto-techref-category/custom-scan-reference.md]: @aklaft  Updated this section based on what I see in the UI: [Additional Configuration](https://664a3ca589a77bb6052686d3--harness-developer.netlify.app/docs/security-testing-orchestration/sto-techref-category/custom-scan-reference#additional-configuration)

File: docs/security-testing-orchestration/sto-techref-category/clair-scanner-reference.md [hunk 12]
```diff
@@ -5,40 +5,121 @@ sidebar_label: Clair scanner reference
...
+##### Key
+```
+product_name
+```
 
-## Security step settings for Clair scans in STO
+##### Value
+
+```yaml
+clair
```
[the remark this answers is not in the document]
Comment (douglas-j-bothwell) [reply docs/security-testing-orchestration/sto-techref-category/clair-scanner-reference.md]: Clair scanner reference > [Scanner settings](https://664a55b389a77bcac8268550--harness-developer.netlify.app/docs/security-testing-orchestration/sto-techref-category/clair-scanner-reference#scanner-settings)

File: docs/security-testing-orchestration/sto-techref-category/custom-scan-reference.md (hunk 8 above)
Comment (douglas-j-bothwell) [reply docs/security-testing-orchestration/sto-techref-category/custom-scan-reference.md]: Did a global update of headings from ### Container images to ### Artifacts

Custom Scan step reference > [Artifacts](https://664a55b389a77bcac8268550--harness-developer.netlify.app/docs/security-testing-orchestration/sto-techref-category/custom-scan-reference#artifacts)

**Pull request merged**